pkg/util/client.go

```diff
@@ -18,20 +18,25 @@ package util
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
 	"io"
+	"io/ioutil"
 	"net/http"
 
-	"io/ioutil"
+	"github.com/Peripli/service-manager/pkg/log"
+)
 
-	"github.com/sirupsen/logrus"
+const (
+	// CorrelationIDHeader is the header for correlation id
+	CorrelationIDHeader = "X-Correlation-ID"
```

Comment:
Already defined in https://github.com/Peripli/service-manager/pull/127/files#diff-5d60679be9a987d5c68b46b7eee08994R26 as [0] element of array.